public/js/eventList.js

```diff
@@ -0,0 +1,36 @@
+
+document.querySelectorAll(".event .lower").forEach(function(event) {
+    event.dataset.height = event.clientHeight
+    event.style.height = 0
+})
+
+document.querySelectorAll(".event").forEach(function(event) {
+    let clicked = 0
+    event.addEventListener('click', function() {
+        clicked++
+        let lower = this.querySelector('.lower')
+        let arrow = this.querySelector(".arrowBox")
+        if(clicked % 2) {
+            arrow.innerHTML = "<span>&#x21E7;</span>"
+
+            var id = setInterval(function() {
```

Comment:
Could you use css transitions instead to handle animation?

Reply:
no because the height is based on the incoming data